src/data/finset/lattice.lean

```diff
@@ -980,6 +980,11 @@ end finset
 
 namespace multiset
 
+lemma map_finset_sup [decidable_eq α] [decidable_eq β]
+  (s : finset γ) (f : γ → multiset β) (g : β → α) (hg : function.injective g) :
+  multiset.map g (s.sup f) = s.sup (multiset.map g ∘ f) :=
```

Comment:
I guess we're in the multiset namespace now, so a bunch of `multiset.`s can be removed from this statement and proof